lib/src/network/coap_network_tinydtls.dart

```diff
@@ -77,7 +77,7 @@ class CoapNetworkUDPTinyDtls extends CoapNetworkUDP {
     super.port,
     super.bindAddress,
     this._tinyDtlsInstance, {
-    super.namespace,
+    final super.namespace,
```

Comment:
I think when I upgraded to Flutter 3.3 (Dart 2.18) or possibly VS Code's Flutter/Dart plugin, a linter rule was added to exclude the need for final super. Maybe we should add it to our lint config too: https://dart.dev/tools/diagnostic-messages?utm_source=dartdev&utm_medium=redir&utm_id=diagcode&utm_content=unnecessary_final#unnecessary_final

Reply:
Thank you! :) I actually haven't had the latest Dart version installed :)